InnerEye/ML/configs/segmentation/HeadAndNeckBase.py

```diff
@@ -115,6 +115,7 @@ def __init__(self,
             largest_connected_component_foreground_classes=ground_truth_ids,
             colours=colours,
             fill_holes=fill_holes,
+            roi_intepreted_types=["ORGAN"] * len(self.fill_holes),
```

Comment:
can we use the same logic as for creating fill_holes? either use constructor arg or autopopulate?